applications/ThermalDEMApplication/custom_constitutive/conduction/direct_conduction_bob_complete.cpp

```diff
@@ -0,0 +1,120 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics ThermalDEM Application
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Rafael Rangel (rrangel@cimne.upc.edu)
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "direct_conduction_bob_complete.h"
+
+namespace Kratos {
+  //-----------------------------------------------------------------------------------------------------------------------
+  DirectConductionBOBComplete::DirectConductionBOBComplete() {}
+  DirectConductionBOBComplete::~DirectConductionBOBComplete() {}
+
+  //------------------------------------------------------------------------------------------------------------
+  double DirectConductionBOBComplete::GetSearchDistance(const ProcessInfo& r_process_info, ThermalSphericParticle* particle) {
+    return particle->GetParticleRadius() * r_process_info[MAX_CONDUCTION_DISTANCE];
+  }
+
+  //------------------------------------------------------------------------------------------------------------
+  double DirectConductionBOBComplete::ComputeHeatFlux(const ProcessInfo& r_process_info, ThermalSphericParticle* particle) {
```

Comment:
just as a minor comment, in order to identify really fast the input variables, they are usually named in CammelCase (rProcessInfo). The local ones are snake_case

Reply:
Ok